### DISCUSSION TITLE
Docs folder getting greyed out in the root directory on VSCode

### DISCUSSION THREAD

**Discussion by TanishqKaushik** [community/community#68365] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Everyone,

I am trying to host my HTML websites generated using .rmd files on VSCode using pages. My question is why is my docs folder getting greyed out every time I create it and not getting recognized by Source Control as something that needs to be committed to the repository?

**Comment by Primus66** [comment, marked as answer]
Check .gitignore File:
	•	Look for a file named .gitignore in your repository’s root directory. This file specifies patterns for files and folders that Git should ignore. Check if “docs” is listed in this file. If it is, remove or modify the entry for “docs” to ensure Git tracks changes in that folder.

Create a File Inside “docs”:
	•	Sometimes, Git might not recognize an empty folder as something to track. Try creating a new file inside the “docs” folder, such as an empty .gitkeep file. This file can serve as a placeholder and ensure that Git tracks the folder.

Commit Changes:
	•	After adding a file to the “docs” folder or modifying the .gitignore file, commit your changes to the repository. This should make Git recognize the “docs” folder and its contents.

Verify Source Control Status:
	•	Check your Source Control panel in VSCode to ensure that the “docs” folder and its files are now recognized and ready to be committed.

By following these steps, you should be able to make Git recognize and track changes in the “docs” folder in your VSCode project, allowing you to host your HTML websites generated from .rmd files using GitHub Pages.

**Reply by TanishqKaushik** [reply]
Thanks for the help @Primus66!

**Reply by Primus66** [reply]
You are welcome!

You could mark my response as answer.😊
